guides/common/modules/ref_smart-proxy-server-scalability-considerations.adoc

```diff
@@ -1,9 +1,7 @@
 [id="{smart-proxy-context}-server-scalability-considerations_{context}"]
-= {SmartProxyServer} scalability considerations
+= {SmartProxyServer} scalability considerations when managing Puppet clients
 
-The maximum number of {SmartProxyServers} that {ProjectServer} can support has no fixed limit.
-It was tested that a {ProjectServer} can support 17 {SmartProxyServers} with 2 vCPUs.
-However, scalability is highly variable, especially when managing Puppet clients.
+{SmartProxyServer} scalability is highly variable, especially when managing Puppet clients.
```

Comment:
I do not understand this sentence.

Reply:
Let's drop it. Looking at the whole section, this sentence doesn't add much value anyway.